setup.cfg

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name = gitserver
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/ceph/gitserver
+author = Dan Mick <dmick@redhat.com>
+license = MIT
+license_file = LICENSE
+classifiers =
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+description_content_type = text/x-rst; charset=UTF-8
+description_file = README.rst
+keywords = git
+summary = A simple Flask-based webserver for querying git commit parent lineage
+
+[options]
+packages = find:
+install_requires =
+    packaging
```

Comment:
Is this for setuptools/toml stuff?  Is it needed for install, or only packaging?

Reply:
It might actually be completely unnecessary; I'll remove it and find out!